django/urls/resolvers.py

```diff
@@ -329,8 +334,22 @@ def check(self):
                     id="2_0.W001",
                 )
             )
+        warnings.extend(self._check_angle_bracket_pattern())
         return warnings
 
+    def _check_angle_bracket_pattern(self):
+        match = re.match(_PATH_ANGLE_BRACKET_COMPONENT_RE, str(self._route))
```

Comment:
Do we need a regular expression? :thinking: Is is not enough to count `<` and `>`.

Reply:
I think it's a solid way for unmatched angle brackets in general. For example:
'closed_angle>/x/<open_angled'

Reply:
Good point :+1: